### DISCUSSION TITLE
Scaling Issue when in "take control"

### DISCUSSION THREAD

**Discussion by rdeanCFS** [amidaware/tacticalrmm#1682] [General]
On v0.17.1 in Chrome, using the take control to remote into a windows machine, it is super zoomed in, cannot see more than like a 3rd of screen, have to manually scale out in the session to 50% and then its only using about half of the actual pop up windows to display the entire screen of the machine being remote in to

**Comment by dinger1986** [comment]
cycle through the views with the far right button 
![image](https://github.com/amidaware/tacticalrmm/assets/7244447/f4e8d3e7-f243-4b99-9703-db2aebd6aeaa)

**Reply by rdeanCFS** [reply]
oh wow thank you
